README.md

```diff
@@ -1,11 +1,97 @@
 # Offender Assessments Events
 
+This service raises business events from OASys for offender assessments via an SNS topic
+
+## Continuous Integration
+
 [![CircleCI](https://circleci.com/gh/ministryofjustice/offender-assessments-events.svg?style=svg)](https://circleci.com/gh/ministryofjustice/offender-assessments-events)
 
-## Running the Application
+### Prerequisites
+* Java JDK 11+
+* An editor/IDE
+* Gradle
+* Docker
+* OAuth  [(running in a container)](#backing-services)
+* LocalStack [(running in a container)](#backing-services)
+
+### Backing services
+In order to run the service locally, [Nomis OAuth Service](https://github.com/ministryofjustice/nomis-oauth2-server/) and [LocalStack](https://github.com/localstack/localstack) are required. This can be run locally using the provided [docker-compose.yaml](docker-compose.yaml) file which will pull down the latest version.
+
+From the command line run:
+```
+ docker-compose up 
+```
+
+### Build service and run tests
+
+This service is built using Gradle. In order to build the project from the command line and run the tests, you require LocalStack to be running, and use:
+```  
+AWS_PROVIDER=localstack AWS_ACCESS_KEY=local AWS_SECRET_KEY=dev ./gradlew clean build 
+```
+The created JAR file will be named "`offender-assessments-events-<yyyy-mm-dd>.jar`", using the date that the build takes place in the format `yyyy-mm-dd`.
+
+
+### Start the application with H2 database
+
+The configuration can be changed for the api to use an in-memory H2 database by using the spring boot profile `dev`. On the command line run:
+```  
+SPRING_PROFILES_ACTIVE=dev 
+java -jar build/libs/offender-assessments-events-<yyyy-mm-dd>.jar  
+```
+
+### Start the application with H2 database and LocalStack configuration
+
+The application can be configured to use LocalStack and an in-memory H2 DB by using the spring boot profile `local`. Ensure you have LocalStack running, on the command line run:
+```  
+SPRING_PROFILES_ACTIVE=dev AWS_PROVIDER=localstack AWS_ACCESS_KEY=local AWS_SECRET_KEY=dev
+java -jar build/libs/offender-assessments-events-<yyyy-mm-dd>.jar  
+```  
```

Comment:
Might need to add to run the localstack script?

Reply:
Good shout, I've added this to the `backing services` section of the document